### PR TITLE
added vip-competence-centre-biomed project

### PR DESCRIPTION
**Pull request by kira2600**
# Summary

---

**Related issue :**

### CODE REVIEW

**Review by enolfc — COMMENTED** [review]
Thanks @kira2600. Can you check the comment?

File: sites/CESNET-MCC.yaml [hunk 1]
```diff
@@ -79,3 +79,6 @@ vos:
 - auth:
     project_id: 426b25102bdb49b8b6969cef5aabaecf
   name: fusion
+- auth:
+    project_id: 721251d671ea4268876043a9474ba3de
+  name: vip-competence-centre-biomed
```
Comment (enolfc) [inline sites/CESNET-MCC.yaml:84]: I missed the comment :)

Is `vip-competence-centre-biomed` a VO? Shouldn't this be `biomed`. If using some subgroup within biomed we need to check how to support that as we didn't do it before.

Comment (kira2600) [reply sites/CESNET-MCC.yaml:84]: Hi Enol,

Information that I got from Mirek that it is some subgroup inside biomed and at our cloud we created separate project for them.

Comment (enolfc) [reply sites/CESNET-MCC.yaml:84]: Ok this is not working so easily. Need to introduce some other changes in the cloud-info-provider to handle the auth so let's keep it as vip-competence-centre-biomed and I will work on fixing it

**Review by enolfc — APPROVED** [review]

**Pull request merged**